home.html

```diff
@@ -5,17 +5,22 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1"/>
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <script src="https://kit.fontawesome.com/c2fdf537fc.js"
-            crossorigin="anonymous"></script>
+
+    <!-- font awesome icons -->
+    <!-- <script src="https://kit.fontawesome.com/c2fdf537fc.js"
+            crossorigin="anonymous"></script> -->
     <title>sona fancy</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"/>
     <!-- favicon -->
     <link rel="icon" href="https://img.icons8.com/flat-round/64/000000/crown--v1.png"/>
+    
     <!-- style 10 component library -->
     <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/padding.css" />
     <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/margin.css" />
     <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/typography.css" />
     <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/card.css" />
     <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/ratings.css" />
```

Comment:
- You can import `main.css` instead of individual imports. Please check if possible.

Reply:
Since I have done some changes in main.css, I import all the required files